### DISCUSSION TITLE
AttributeError: module 'demo' has no attribute 'app'

### DISCUSSION THREAD

**Discussion by TatianaLeal1125** [aiortc/aioquic#344] [Q&A]
Hello,

I am trying to run the http3_server.py in my local machine, but I got this error:

application = getattr(module, attr_str)
AttributeError: module 'demo' has no attribute 'app'

I really have the server and demo script in my computer. What can I do to resolve this error?

**Comment by devkdas** [comment]
Hope you didnt made any changes to your local aioquic project files

root@ubuntu:~/aioquic# python examples/http3_server.py --certificate tests/ssl_cert.pem --private-key tests/ssl_key.pem -v
Traceback (most recent call last):
  File "examples/http3_server.py", line 562, in <module>
    application = getattr(module, attr_str)
AttributeError: module 'demo' has no attribute 'app'

Hope you are getting same above error .

if so make sure you have 


async def app(scope: Scope, receive: Receive, send: Send) -> None:
    if scope["type"] == "webtransport" and scope["path"] == "/wt":
        await wt(scope, receive, send)
    else:
        await starlette(scope, receive, send)  

in demo.py

Hope this helps

**Comment by YifangZhang** [comment]
It could happen if you are not performing the http3_server.py in the example folder. As long as you moved the demo.py with your new destination, you should not see the same error anymore.
